docs/quick_start.md

```diff
@@ -29,6 +29,43 @@ Paste the following in a macOS Terminal and press enter to install Homebrew.
 Note: *When this command is run, additional installations may be automatically added by Homebrew based on your machine's configurations.
```

Comment:
Can you make sure you merge your branch with main? There's some merge conflicts and there should definitely be some information on Windows users here

Reply:
this branch was created this morning with the updated main before merged your PR

Reply:
yes I will fix the conflicts